dbt/adapters/presto/connections.py

```diff
@@ -178,7 +179,7 @@ def open(cls, connection):
         return connection
 
     @classmethod
-    def get_status(cls, cursor):
+    def get_response(cls, cursor):
```

Comment:
> this is lame, but the cursor doesn't give us anything useful.

This is indeed lame, and it means we don't have anything nice to populate in `adapter_response`.

The same is true of `dbt-spark` today ([link](https://github.com/fishtown-analytics/dbt-spark/blob/2f7c2dcb419b66d463bb83212287bd29280d6964/dbt/adapters/spark/connections.py#L275-L277)), though I do wonder if it varies by connection type? Is there a chance that the `PyodbcConnectionWrapper` returns us richer info than the `PyhiveConnectionWrapper`?

Reply:
Good call! Yes, I believe you are are right. I will confirm as well